src/schema/common.ts

```diff
@@ -0,0 +1,92 @@
+export interface ObjectSchema extends Record<string, unknown> {
```

Comment:
There is a great deal of schemas in this file that exist in page-constructor 
For example: https://github.com/gravity-ui/page-constructor/blob/9a1522d2eec7435c1431fd6b28bc428e154695cb/src/schema/validators/common.ts#L23

Is there any reason to add it here instead of import from PC?

Reply:
fix